### DISCUSSION TITLE
Shulker, toggle to disable breeding

### DISCUSSION THREAD

**Discussion by TomLewis** [PurpurMC/Purpur#885] [__Completed__] [answered]
in Purpur configs under shulker could we please have a toggle to disable their breeding. 👍 

Then you can control the use of shulker boxes on a server.

**Comment by BillyGalbreath** [comment, marked as answer]
Shulker's don't really "breed" but they duplicate when hit by a shulker bullet. We already have configuration for that.

https://purpurmc.org/docs/Configuration/#base-chance

Just set `base-chance` to `0` if you want to disable that feature.

**Comment by TomLewis** [comment]
Aha you are the man @BillyGalbreath! 
I was thinking along the wrong lines of what it would be called!

I will enable this now, thank you.
